<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1322</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Silla</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mesa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pizarra</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Librero</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Buro</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboMes">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboMaterial">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Aluminio</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Madera</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboLugar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Departamento</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aula</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="anchoL">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancho</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="anchoLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="largoLine">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="largoL">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Largo</string>
    </property>
   </widget>
   <widget class="QLabel" name="altoL">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>380</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Alto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="altoLine">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cantidadPatasLine">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de patas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cantidadPatasL">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cantidadMaterialLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cantiMaterialL">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad Material</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mueble</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Material</string>
    </property>
   </widget>
   <widget class="QLabel" name="tipoSillaL">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de silla</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tipoSillaLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>400</y>
      <width>113</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lugar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="paletaCheck">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>93</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Paleta</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPromedioM">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Promedio Mensual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="mesLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mes de mas mesas</string>
    </property>
   </widget>
   <widget class="QLabel" name="excedioPromedio">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>200</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Producto excedio promedio</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="semestreCheck">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Primer Semestre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>240</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="listado">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>250</y>
      <width>291</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Producto excedio promedio</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1322</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
